CONTRIBUTING.md

```diff
@@ -82,17 +83,26 @@ Your pull request should be rebased against the current master branch. Please do
 the current master branch in with your topic branch, nor use the Update Branch button provided
 by GitHub on the pull request page.
 
+## Submitting a Plugin
+
+Sharing a new plugin is one of the most impressive ways to jump into the Snap community as a contributor. This process is as simple as adding your plugin information to the [list in plugis.yml](docs/plugin.yml).
+
+All plugins mature at their own pace and have varying levels of support from maintainers. Each status and its corresponding requirements are available in the [Plugin Status documentation](docs/PLUGIN_STATUS.md).
```

Comment:
I don't think this is the right place to mention plugin status. I see it as something people must do for their plugin to be considered Supported. They will see this requirement and others when they go to Plugin_Authoring.md.